CHANGELOG.md

```diff
@@ -1,4 +1,42 @@
-# safe-nd - Change Log
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### 0.8.1 (2020-07-20)
+
+
+### Features
+
+* **audit:** add scheduled security audit scan ([4725d13](https://github.com/joshuef/safe-nd/commit/4725d13ee7a473f46bf2e119d5021b6ab5f8fb71))
+* **bump:** test GHA ([8142f8a](https://github.com/joshuef/safe-nd/commit/8142f8ad137a38f43ba43d33bb46e6f1f320b950))
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
+
+### 0.8.1 (2020-07-20)
+
+
+### Features
+
+* **audit:** add scheduled security audit scan ([4725d13](https://github.com/joshuef/safe-nd/commit/4725d13ee7a473f46bf2e119d5021b6ab5f8fb71))
+* **bump:** test GHA ([8142f8a](https://github.com/joshuef/safe-nd/commit/8142f8ad137a38f43ba43d33bb46e6f1f320b950))
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
+
+### 0.8.1 (2020-07-20)
+
+
+### Features
+
+* **audit:** add scheduled security audit scan ([4725d13](https://github.com/joshuef/safe-nd/commit/4725d13ee7a473f46bf2e119d5021b6ab5f8fb71))
+* **bump:** test GHA ([8142f8a](https://github.com/joshuef/safe-nd/commit/8142f8ad137a38f43ba43d33bb46e6f1f320b950))
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
+
+### 0.8.1 (2020-07-20)
+
+
+### Features
+
+* **audit:** add scheduled security audit scan ([4725d13](https://github.com/joshuef/safe-nd/commit/4725d13ee7a473f46bf2e119d5021b6ab5f8fb71))
+* **bump:** test GHA ([8142f8a](https://github.com/joshuef/safe-nd/commit/8142f8ad137a38f43ba43d33bb46e6f1f320b950))
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
 
 ## [0.10.1]
```

Comment:
0.10.1, 0.10.0 and 0.9.0 should be above 0.8.1. Is this because 0.8.1 was missing from the changelog?

Reply:
ah, all that needs cleaned out ups. 

That's me having run the action a bunch of times w/ different versions and forgetting to clean the changelog :|

Reply:
(and my repo not having latest tags, so opting for 8.1)